docs/single-spa-api.md

```diff
@@ -108,6 +108,9 @@ The `fn` must be a function. It will be called with one argument, `err`, which i
 ## removeErrorHandler
 `removeErrorHandler(fn)` removes an error handler. Returns true if the error handler was removed, and false if it was not.
 
+## mountRootParcel
+`mountRootParcel(parcelConfig)` will create and mount a parcel. The parcel will not be automatically unmounted and unmounting will need to be triggered manually.
```

Comment:
This is missing the props argument. Including the required `domElement` prop

Reply:
Additionally, it should probably be explained that parcelConfig can optionally be a loading function that returns a Promise that resolves with the parcel config.